frontend/app/views/giraffe/contribute.scala.html

```diff
@@ -207,7 +208,10 @@ <h3 class="cvc__heading">American Express</h3>
 
                     <div class="fieldset__note">
                         <p>By proceeding, you are agreeing to our
-                            <a href="@Links.giraffeTerms" class="text-link" target="_blank">Terms and Conditions</a> and
+                            <a href="@Links.giraffeTerms" class="text-link" target="_blank" data-shown="gbp">Terms and Conditions</a>
+                            <a href="@Links.giraffeTermsAustralia" class="text-link" target="_blank" data-shown="aud">Terms and Conditions</a>
```

Comment:
It should be possible to pass this into the view rather than selecting it in the template using CSS etc.  I.e. Add to PageInfo etc for given country.